.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-        "source.fixAll": true,
-        "source.organizeImports": true,
+        "source.fixAll": "explicit",
+        "source.organizeImports": "explicit"
```

Comment:
Visual Studio Code did this by itself:
![image](https://github.com/nullxception/boorusphere/assets/49622129/72d7ac7a-e440-40a6-a350-0540c8369dba)